front50-gcs/src/main/java/com/netflix/spinnaker/front50/model/GcsStorageService.java

```diff
@@ -187,6 +187,7 @@ public void ensureBucketExists() {
   /**
    * Returns true if the storage service supports versioning.
    */
+  @Override
```

Comment:
I don't believe I changed the functionality of anything in here but I'll need you to give it a whirl once things are closer to landing (probably tomorrow sometime).

This differs slightly from `s3` due to the difference in naming around filenames (`specification.json` vs `application-metadata.json` and `last-modified.json` vs. `last-modified`).

Also added missing @Override's